load_agent.py

```diff
@@ -66,3 +66,6 @@ def episode(env, agent):
             print("Done!")
     print('test episode:', 'R:', R)
     agent.stop_episode()
+
+
+create_environment("../dataset-generator/image_locations.txt", "../dataset-generator/bounding_boxes.npy")
```

Comment:
Are you sure that this should be checked in? If yes, should it maybe be wrapped in the following way?

```suggestion
if __name__ == '__main__':
    create_environment("../dataset-generator/image_locations.txt", "../dataset-generator/bounding_boxes.npy")
```

Reply:
ops I didn't even see that change... It wasn't meant to be there I simply used it for trying out the code. Gonna take it away. 

P.S.
I wouldn't put anything in there because load_agent in general should probably be used by anyone according to their needs... it doesn't really have a typical use scenario.